src/ray/object_manager/push_manager.cc

```diff
@@ -72,4 +75,20 @@ void PushManager::ScheduleRemainingPushes() {
   }
 }
 
+void PushManager::RecordMetrics() const {
+  ray::stats::STATS_num_pushes_in_flight.Record(NumPushesInFlight());
+  ray::stats::STATS_num_chunks_in_flight.Record(NumChunksInFlight());
+  ray::stats::STATS_num_chunks_remainig.Record(NumChunksRemaining());
```

Comment:
```suggestion
  ray::stats::STATS_num_chunks_remaining.Record(NumChunksRemaining());
```

Also, should consolidate and use labels for breakdown.